primitive/message.go

```diff
@@ -69,6 +69,7 @@ const (
 )
 
 type Message struct {
+	List           []*Message
```

Comment:
the list is used internal, should not export to public interface. so might be better to pass msgs to ProcessSendResponse function through function call.